tests/testthat/test_civis_ml_utils.R

```diff
@@ -37,7 +35,8 @@ test_that("print.civis_ml_regressor works", {
 
   third_row <- lapply(reg_msg, purrr::pluck, 3)
   expect_true(all(str_detect_multiple(third_row, id_regex[!is_classif])))
-  expect_true(all(stringr::str_detect(reg_msg, "(MAD)*(RMSE)*(R-squared)")))
+
+  expect_true(all(stringr::str_detect(reg_msg[1:5], "(MAD)*(RMSE)*(R-squared)")))
```

Comment:
What is this change doing? I'm assuming this is test cleanup that's not related to CivisML 2.0?